.github/workflows/axosyslog-docker.yml

```diff
@@ -151,11 +153,30 @@ jobs:
 
       - name: Extract metadata (AxoSyslog version) for Docker
         id: docker-metadata-tags
-        uses: docker/metadata-action@v5
-        with:
-          images: ${{ needs.prepare.outputs.image-name }}
-          tags: type=match,pattern=axosyslog-([0-9].*),group=1
-          sep-tags: ','
+        run: |
+          previous_docker_tag=$(skopeo inspect docker://ghcr.io/axoflow/axosyslog:latest | jq '.RepoTags' | sed 's/ \|"\|,//g' | grep "^[0-9]*.[0-9]*.[0-9]*$\|^[0-9]*.[0-9]*.[0-9]*-[0-9]*$" | tail -n 1)
+          previous_docker_tag_short=$(echo $previous_docker_tag | cut -d"-" -f1)
+          previous_docker_tag_patch=$(echo $previous_docker_tag | cut -d"-" -f2)
+          current_git_tag=$(echo ${{ github.ref }} | cut -d"-" -f2)
+          function version { echo "$@" | awk -F. '{ printf("%d%03d%03d\n", $1,$2,$3); }'; }
+          if [ $(version $current_git_tag) -gt $(version $previous_docker_tag_short) ]; then
+              echo "Given git tag version is greater than previous docker tag version: $current_git_tag > $previous_docker_tag_short"
+              next_docker_tag_with_patch=$current_git_tag-1
+              next_docker_tag=$current_git_tag
+              echo "Next docker version tag: $next_docker_tag_with_patch"
+          elif [ $(version $current_git_tag) -eq $(version $previous_docker_tag_short) ]; then
+              echo "Given git tag version is equals with previous docker tag version: $current_git_tag == $previous_docker_tag_short"
```

Comment:
`AxoSyslog image rebuild detected (version: )`

Reply:
Thanks, I have fixed the info messages in the job